### DISCUSSION TITLE
Initial Feedback

### DISCUSSION THREAD

**Discussion by chrisjsewell** [aiidateam/aiida-firecrest#2] [General]
Please leave any feedback here!

**Comment by ltalirz** [comment]
register on https://github.com/aiidateam/aiida-registry ? ;-)

**Reply by chrisjsewell** [reply]
yeh will do a bit more work first 😅
was also meaning to see if I can get the registry to work with `setup.cfg` and `version = attr: aiida_firecrest.__version__`, that would be ideal to not have to use the bespoke `setup.json`

**Comment by giovannipizzi** [comment]
Thanks Chris for your work on this already! Happy to jump into a discussion on this later next week!

**Comment by chrisjsewell** [comment]
Note, I think this endpoint addition would certainly be very desirable for this plugin: https://github.com/eth-cscs/firecrest/pull/133

**Comment by espenfl** [comment]
Really great. Thanks a lot for the effort @chrisjsewell. Could we possibly start some kind of track to utilize `libsubmit` or somehow contribute to something similar such that we in the future could have a library to handle scheduler interfaces more broadly?

**Reply by chrisjsewell** [reply]
Thanks @espenfl, can you briefly explain what `libsubmit` is (I haven't heard of it before 😬 )

**Reply by chrisjsewell** [reply]
Ah I'm guessing you mean this: https://github.com/Parsl/libsubmit

**Reply by espenfl** [reply]
But concept being more of a library that handles the most common schedulers. Would make sense for e.g. FirecREST to rely on something like that. Also AiIDA. Maybe we can start a joint effort on such a thing? We can also air this with the developers for Parsl and try to gather the developers.
